src/to-fresh.js

```diff
@@ -71,6 +71,7 @@ Convert JRS resume sections to FRESH.
             start: edu.startDate,
             end: edu.endDate,
             grade: edu.gpa,
+            title: edu.title,
```

Comment:
Here's some more information on the FRESH title field: https://github.com/fresh-standard/FRESCA/issues/10